mhs/main.py

```diff
@@ -44,39 +43,37 @@ def load_party_key(data_dir: pathlib.Path) -> str:
     return party_key
 
 
-def initialise_workflow(data_dir: pathlib.Path, certs_dir: pathlib.Path,
-                        party_key: str) -> sync_async_workflow.SyncAsyncWorkflow:
-    """Initialise the
+def initialise_workflows(certs_dir: pathlib.Path, party_key: str) -> Dict[str, workflow.CommonWorkflow]:
+    """Initialise the workflows
 
-    :param data_dir: The directory to load interactions configuration from.
     :param certs_dir: The directory containing certificates/keys to be used to identify this MHS to a remote MHS.
     :param party_key: The party key to use to identify this MHS.
-    :return: The workflow that can be used to handle messages.
+    :return: The workflows that can be used to handle messages.
     """
-    interactions_config_file = str(data_dir / "interactions" / "interactions.json")
-    config_manager = configuration_manager.ConfigurationManager(interactions_config_file)
+    # transmission = outbound_transmission.OutboundTransmission(str(certs_dir))
+    # workflow = sync_async_workflow.SyncAsyncWorkflow(transmission, party_key)
```

Comment:
Reason I left them was because the next task, working on the workflow, would mean that we'd essentially bung some of this code back in again. I know, it's not amazing committing commented-out code, but it felt a bit simpler than just stripping the code out only to add it in again in a pretty similar fashion.